packages/renderer/src/lib/image/RunImage.svelte

```diff
@@ -69,68 +170,661 @@ async function startContainer() {
+                {#each exposedPorts as port, index}
+                  <div class="flex flex-row justify-center items-center w-full">
+                    <span class="text-sm flex-1 inline-block align-middle whitespace-nowrap text-gray-400"
+                      >Local port for {port}:</span>
+                    <input
+                      type="text"
+                      bind:value="{containerPortMapping[index]}"
+                      placeholder="Enter value for port {port}"
+                      class="ml-2 w-full p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400" />
+                  </div>
+                {/each}
+
+                <label
+                  for="modalEnvironmentVariables"
+                  class="pt-4 block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Environment variables:</label>
+                <!-- Display the list of existing environment variables -->
+                {#each environmentVariables as environmentVariable, index}
+                  <div class="flex flex-row justify-center items-center w-full py-1">
+                    <input
+                      type="text"
+                      bind:value="{environmentVariable.key}"
+                      placeholder="Enter a Name"
+                      class="ml-2 w-full p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400" />
+
+                    <input
+                      type="text"
+                      bind:value="{environmentVariable.value}"
+                      placeholder="Value (or leave blank for empty)"
+                      class="ml-2 w-full p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400" />
+                    <button
+                      class="ml-2 p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400"
+                      hidden="{index === environmentVariables.length - 1}"
+                      on:click="{() => deleteEnvVariable(index)}">
+                      <Fa class="h-4 w-4 text-xl" icon="{faMinusCircle}" />
+                    </button>
+                    <button
+                      class="ml-2 p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400"
+                      hidden="{index < environmentVariables.length - 1}"
+                      on:click="{addEnvVariable}">
+                      <Fa class="h-4 w-4 text-xl" icon="{faPlusCircle}" />
+                    </button>
+                  </div>
+                {/each}
+              </div>
+            </Route>
+            <Route path="/advanced">
+              <div class="h-96 overflow-y-auto pr-4">
+                <!-- Use tty -->
+                <label for="containerTty" class="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Use tty:</label>
```

Comment:
```suggestion
                  >Use TTY:</label>
```

TTY is an acronym and should be capitalized.